src/frontend/src/error.rs

```diff
@@ -114,7 +114,7 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
-    #[snafu(display("Table not found: {}", table_name))]
+    #[snafu(display("Table `{}` not exist", table_name))]
```

Comment:
Why change this error? Looks like it's not necessary. And if we want to quote the table name or schema name, I think we should change all error messages too.

Reply:
Change the error's display message, to comply with the expected output originally written in standalone test case.